src/content/api/loaders.md

```diff
@@ -471,3 +509,95 @@ Hacky access to the Compiler object of webpack.
 ### `this._module`
 
 Hacky access to the Module object being loaded.
+
+
+## Error Reporting
+
+Here are the ways to throw an error from inside a loader:
+
+### Using `this.emitError`
+
+[this.emitError](/api/loaders/#this-emiterror) will report the errors without interrupting module's compilation.
+
+
+### Using `throw` (or other uncaught exception)
+
+Throwing an error while a loader is running will cause current module compilation failure.
+
+For example:
+
+__./src/index.js__
+
+```js
+
+require('./loader!./lib');
+
+```
+
+__./src/loader.js__
+
+```js
+
+module.exports = function(source){
+  throw new Error('Here is an fatal Error!');
+};
+
+```
+
+The module on which loader had thrown this error will get bundled like this:
+
+```js-with-links
```

Comment:
do we still support js-with-links?

Reply:
cc @montogeek

Reply:
Yes

Reply:
This is master, it should work

Reply:
But you are not using them, `js` should be enough. Please change it.

Reply:
ok so there is only 1 outstanding issue i see with pr, compiling -> compilation

Reply:
> But you are not using them, js should be enough. Please change it.

Here I want to display the original bundle code. but If I use `js`, the linter will scan the code and request for the fix. Could I ignore the linter here?

Reply:
What it the linter error? You could disable it.